lib/copyright-exceptions.json

```diff
@@ -31,5 +31,9 @@
     {
         "specShortnames": ["sdw-bp"],
         "copyright": "Copyright © @YEAR <a href=\"https://www.ogc.org/\">OGC</a> &amp; <a href=\"https://www.w3.org/\">World Wide Web Consortium</a>. <abbr title=\"World Wide Web Consortium\">W3C</abbr><sup>®</sup> <a href=\"https://www.w3.org/policies/#Legal_Disclaimer\">liability</a>, <a href=\"https://www.w3.org/policies/#W3C_Trademarks\">trademark</a>, <a href=\"https://www.w3.org/copyright/document-license/\">W3C</a> and <a href=\"https://www.ogc.org/ogc/document\">OGC</a> document use rules apply."
+    },
+    {
+        "specShortnames": ["html-ruby-extensions"],
```

Comment:
I heard the shortname for the spec hasn't been decided yet. Are we going with `html-ruby-extensions`?

Reply:
Right. The WG hasn't made any resolutions yet about publication, so there's not resolution about any particular shortname. But I think this is most likely the one we're going to go with:
* the spec is a continuation of https://www.w3.org/TR/html-ruby-extensions/, both in spirit, and in terms of content
* the charter, [in its list of deliverables](https://www.w3.org/2022/06/html-wg-charter.html#ruby),  does point to https://www.w3.org/TR/html-ruby-extensions/ when describing this deliverable, including linking to https://www.w3.org/TR/2013/WD-html-ruby-extensions-20131022/ as the Adopted Draft / Exclusion Draft
* Keeping it as the same shortname helps make the continuity clear, which is nice in general, and in particular for the point of view of the Patent Policy.

So it might en up changing if for some reason the WG resolves differently, but I think it's a reasonable assumption to that `html-ruby-extensions` is what it'll be.

Reply:
Thanks for the details. Note that the shortname can still be changed later on.